catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpTransformer.java

```diff
@@ -0,0 +1,604 @@
+        addValuesFromPathToListIfPresent(pathValueTracker,
+                isrCategory,
+                MgmpConstants.ISR_COVERAGE_CATEGORY_PATH,
+                false);
+        addValuesFromPathToListIfPresent(pathValueTracker,
+                isrCategory,
+                MgmpConstants.ISR_IMAGE_DESCRIPTION_PATH,
+                false);
+        addValuesFromPathToListIfPresent(pathValueTracker,
+                isrCategory,
+                MgmpConstants.ISR_MD_IMAGE_DESCRIPTION_PATH,
+                false);
+        addValuesFromPathToListIfPresent(pathValueTracker,
+                isrCategory,
+                MgmpConstants.ISR_VIDEO_DESCRIPTION_PATH,
+                false);
+
+        metacard.setAttribute(Isr.COMMENTS, (Serializable) isrComments);
+        metacard.setAttribute(Isr.CATEGORY, (Serializable) isrCategory);
+
+        addMetacardIsrOrganizationalUnit(metacard);
+    }
+
+    public void addMetacardIsrOrganizationalUnit(MetacardImpl metacard) {
+        Attribute pocName = metacard.getAttribute(Contact.POINT_OF_CONTACT_NAME);
+        Attribute publisherName = metacard.getAttribute(Contact.PUBLISHER_NAME);
+
+        if (pocName != null) {
+            setIsrOrganizationalUnitFromAttribute(pocName, metacard);
+        } else if (publisherName != null) {
+            setIsrOrganizationalUnitFromAttribute(publisherName, metacard);
+        }
+    }
+
+    private void setIsrOrganizationalUnitFromAttribute(Attribute pocName, MetacardImpl metacard) {
+        Serializable pointOfContact = pocName.getValue();
+        if (pointOfContact instanceof String) {
+            String pocValue = (String) pointOfContact;
+            if (StringUtils.isNotEmpty(pocValue)) {
+                metacard.setAttribute(Isr.ORGANIZATIONAL_UNIT, pocValue);
+            }
+        }
+    }
+
+    private void addMetacardNiirsInformation(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        List<String> ratingSystemList =
+                pathValueTracker.getAllValues(toPath(MgmpConstants.NIIRS_PATH));
+        List<String> ratingsList =
+                pathValueTracker.getAllValues(toPath(MgmpConstants.NIIRS_RATING_PATH));
+        if (CollectionUtils.isNotEmpty(ratingsList)
+                && CollectionUtils.isNotEmpty(ratingSystemList)) {
+            for (int i = 0; i < ratingsList.size() && i < ratingSystemList.size(); i++) {
+                if (ratingSystemList.get(i)
+                        .equals(MgmpConstants.NIIRS)) {
+                    try {
+                        Double niirsRating = Double.parseDouble(ratingsList.get(i));
+                        metacard.setAttribute(Isr.NATIONAL_IMAGERY_INTERPRETABILITY_RATING_SCALE,
+                                niirsRating);
+                    } catch (NumberFormatException e) {
+                        LOGGER.debug("Unable to parse double when setting NIIRS value", e);
+                    } finally {
```

Comment:
There's no reason to put this break inside this `finally`. If any other runtime exception did occur, you'd break all the way out of this loop (and call stack) anyway.